### PR TITLE
More explicit objectives for raiders

### LINKED ISSUE #25024: Pox Raiders are not informed of what they need to steal
I don't know. It looks like the contents of the `{high,medium,low}_score_items` lists are only used to check whether they stole the thing, and never to actually tell them to do it.

The jectie just says "Acquire [x] priority items".

### PR DESCRIPTION
**Pull request by ShiftyRail**
[bugfix]
Closes #25024, at least partly. I'll add something to analyse the value of an item in another PR.

### CODE REVIEW

File: code/datums/gamemode/factions/vox_shoal.dm [hunk 1]
```diff
@@ -106,14 +106,16 @@ var/list/potential_bonus_items = list(
 
 	for (var/i = 1 to 4)
 		var/chosen_one = pick(potential_bonus_items_temp)
-		potential_bonus_items_temp =- chosen_one
+		to_chat(world, "[chosen_one]")
```
Comment (DamianX) [inline code/datums/gamemode/factions/vox_shoal.dm]: Auf der Heide blüht ein kleines Blümelein

File: code/datums/gamemode/factions/vox_shoal.dm [hunk 2]
```diff
@@ -106,14 +106,15 @@ var/list/potential_bonus_items = list(
 
 	for (var/i = 1 to 4)
 		var/chosen_one = pick(potential_bonus_items_temp)
-		potential_bonus_items_temp =- chosen_one
```
Comment (Exxion) [inline code/datums/gamemode/factions/vox_shoal.dm:109]: lol

File: code/datums/gamemode/factions/vox_shoal.dm [hunk 3]
```diff
@@ -106,14 +106,15 @@ var/list/potential_bonus_items = list(
...
+		potential_bonus_items_temp.Remove(chosen_one)
```
Comment (Exxion) [inline code/datums/gamemode/factions/vox_shoal.dm]: Not that it actually matters, but generally `-=` is preferred to `Remove()`

File: code/datums/gamemode/factions/vox_shoal.dm (hunk 2 above)
Comment (Zth--) [reply code/datums/gamemode/factions/vox_shoal.dm:109]: this is funny. So I did a grep to see if this happened somewhere else and the answer is yes:

https://github.com/vgstation-coders/vgstation13/blob/94e4ee25da2bbd1eb53aff5a4a26d0b19cfda7e6/code/modules/events/meteors.dm#L234

File: code/datums/gamemode/factions/vox_shoal.dm (hunk 3 above)
Comment (ShiftyRail) [reply code/datums/gamemode/factions/vox_shoal.dm]: Ignore the previous comment, you're right

File: code/datums/gamemode/factions/vox_shoal.dm (hunk 2 above)
Comment (Exxion) [reply code/datums/gamemode/factions/vox_shoal.dm:109]: This should really runtime anytime the var on the right isn't a number
I haven't checked, though, so maybe it does

EDIT: It does not runtime. Apparently it returns 0.

**Pull request merged**